services/manage/app/controllers/manage/orders/edit/lineitems/edit/index.js

```diff
@@ -112,6 +120,26 @@ export default Controller.extend(ObjectQueryManager, ActionMixin, {
       }
     },
 
+    /**
+     *
+     */
+    async pause() {
+      this.startAction();
+      this.set('isPausing', true);
+      const id = this.get('model.id');
+      const value = !this.get('model.paused');
+      const variables = { input: { id, value } };
+      const mutation = pauseLineItem;
+      try {
+        await this.get('apollo').mutate({ mutation, variables }, 'pauseLineItem');
+      } catch (e) {
+        this.get('graphErrors').show(e);
+      } finally {
+        this.endAction();
+        this.set('isPausing', false);
```

Comment:
The `isPausing` would not be necessary if you make the switch auto save component disable itself on save